### ISSUE TITLE
Imgur video thumbnails

### ISSUE THREAD

**Issue by K4LCIFER** [thunder-app/thunder#1420]
### Is Your Feature Request Related to a Problem?

I've seen imgur videos without thumbnails

### Feature Description

Provide thumbnails for Imgur videos

### Additional Context

I'm not sure if this is an issue with all Imgur videos, or just failings for the one's that I have seen. Here is an example of one:
![Screenshot_20240604-155544](https://github.com/thunder-app/thunder/assets/7829231/3b13d55e-2002-4441-982d-57cd1ce1cfcc)

**Comment by micahmo** [comment]
Hey @K4LCIFER, thanks for the report! We generally rely on Lemmy to generate thumbnails for us as it can be very expensive for us to scrape websites and generate thumbnails from within Thunder. Can you see whether this post has a thumbnail on the web UI?

**Comment by K4LCIFER** [comment]
> We generally rely on Lemmy to generate thumbnails for us as it can be very expensive for us to scrape websites and generate thumbnails from within Thunder.

Noted!

---

> Can you see whether this post has a thumbnail on the web UI?

It does not.
![image](https://github.com/thunder-app/thunder/assets/7829231/1697cdea-587e-4675-b9dc-2b022c8e7057)

---

I shall close this issue then! Out of scope.

**Comment by micahmo** [comment]
You can possibly try the "Scrape Missing Link Previews" setting, but there's no guarantee it'll help. That's our backup for when Lemmy doesn't have a thumbnail, but it's off by default due to the extra overhead as mentioned.

**Comment by micahmo** [comment]
By the way, don't feel like you have to close these kinds of issues if they're not resolved for you. 😊 I'm just trying to be upfront about how (or if) we would possibly tackle this from our end.

**Comment by K4LCIFER** [comment]
> By the way, don't feel like you have to close these kinds of issues if they're not resolved for you. 😊 I'm just trying to be upfront about how (or if) we would possibly tackle this from our end.

I appreciate it! Perhaps some tag like "on-hold" or "upstream" would be good if it is kept open?

**Comment by micahmo** [comment]
Good idea, we have several issues that rely on Lemmy server-side changes, so I think I'll make a corresponding label!
